### ISSUE TITLE
Failed to load IMU data

### ISSUE THREAD

**Issue by qian-chu** [pupil-labs/pl-neon-recording#21]
I'm playing with NeonRecording class with the sample dataset (from https://api.cloud.pupil-labs.com/v2/workspaces/3d240903-cbf7-485a-84b1-3498f00e236c/recordings.zip?id=7c141c32-7a8d-411c-9428-70d16de9a9ea&share-key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiJmY2Q5NWJkYS1lZjVhLTRlN2YtOWZiYi01NTc1NWFhYWQzZTAiLCJtZXRob2QiOiJHRVQiLCJwYXRoIjoiL3YyL3dvcmtzcGFjZXMvM2QyNDA5MDMtY2JmNy00ODVhLTg0YjEtMzQ5OGYwMGUyMzZjL3JlY29yZGluZ3MuemlwIiwicXVlcnkiOiJpZD03YzE0MWMzMi03YThkLTQxMWMtOTQyOC03MGQxNmRlOWE5ZWEiLCJleHAiOjE4NjkyMjA3NzkuMjEzMDEzNn0.xfR9ZqOWmql1EmAkqc_mszL63hER_s8HziAzfn89-QI) and a minimalist code:
```
from pupil_labs.neon_recording import load
recording = load(data_path)
print(recording.imu)
```
I got the following error:
```
C:\Users\qian.chu\AppData\Local\anaconda3\envs\neon2bids\Lib\site-packages\pupil_labs\neon_recording\stream\imu\imu_stream.py:121: RuntimeWarning: overflow encountered in scalar add
  packet_bytes = buffer[index: index + packet_size]
C:\Users\qian.chu\AppData\Local\anaconda3\envs\neon2bids\Lib\site-packages\pupil_labs\neon_recording\stream\imu\imu_stream.py:122: RuntimeWarning: overflow encountered in scalar add
  index += packet_size
Traceback (most recent call last):
  File "c:\Users\qian.chu\Documents\GitHub\neon2bids\example_dataset\playground.py", line 14, in <module>
    print(recording.imu)
          ^^^^^^^^^^^^^
  File "C:\Users\qian.chu\AppData\Local\anaconda3\envs\neon2bids\Lib\site-packages\pupil_labs\neon_recording\neon_recording.py", line 102, in imu
    self.streams["imu"] = IMUStream(self)
                          ^^^^^^^^^^^^^^^
  File "C:\Users\qian.chu\AppData\Local\anaconda3\envs\neon2bids\Lib\site-packages\pupil_labs\neon_recording\stream\imu\imu_stream.py", line 87, in __init__    
    rotation = Rotation.from_quat(
               ^^^^^^^^^^^^^^^^^^^
  File "_rotation.pyx", line 997, in scipy.spatial.transform._rotation.Rotation.from_quat
  File "_rotation.pyx", line 855, in scipy.spatial.transform._rotation.Rotation.__init__
ValueError: Found zero norm quaternions in `quat`.
```

**Comment by domstoppable** [comment]
Thanks for the report, @qian-chu - we're investigating this to determine the cause of the bad quats (so that future recordings don't have them) and deciding how to handle them in existing recordings

**Comment by domstoppable** [comment]
Fixed in c55f5b9
